packages/react/src/components/Alert/Alert.tsx

```diff
@@ -53,35 +20,19 @@ export type AlertProps = {
 /**
  * Alerts are used to inform users about important information, warnings, errors, or success.
  * @example
- * <Alert severity='info'>Dette er en informasjonsmelding</Alert>
+ * <Alert color='info'>Dette er en informasjonsmelding</Alert>
  */
-export const Alert = forwardRef<HTMLDivElement, AlertProps>(
-  (
-    { severity = 'info', iconTitle, children, size = 'md', className, ...rest },
-    ref,
-  ) => {
-    const { Icon, title } = icons[severity];
-
-    return (
-      <div
-        ref={ref}
-        className={cl(
-          'ds-alert',
-          `ds-alert--${size}`,
-          `ds-alert--${severity}`,
-          className,
-        )}
-        {...rest}
-      >
-        <>
-          <Icon title={iconTitle || title} className='ds-alert__icon' />
-          <Paragraph asChild size={size} className='ds-alert__content'>
-            <span>{children}</span>
-          </Paragraph>
```

Comment:
ref comment above, we should keep `asChild` until we can compose

Reply:
Will fix - dette var nok en misforståelse fra sist møte :)